src/WebJobs.Script/Eventing/ScriptEventManager.cs

```diff
@@ -11,7 +11,13 @@ public sealed class ScriptEventManager : IScriptEventManager, IDisposable
         private readonly Subject<ScriptEvent> _subject = new Subject<ScriptEvent>();
         private bool _disposed = false;
 
-        public void Publish(ScriptEvent scriptEvent) => _subject.OnNext(scriptEvent);
+        public void Publish(ScriptEvent scriptEvent)
+        {
+            if (!_disposed)
```

Comment:
Yeah. This does point to an issue where disposed resources continue to be used by other types, and this change may hide some of this issues. Ideally, we'd identify where they're being misused and fix that instead. Thoughts?